docs/README.md

```diff
@@ -0,0 +1,10 @@
+# Mapzen Terrain Tile documentation
+The [Mapzen terrain tiles](https://mapzen.com/projects/joerd) provide basemap terrain coverage of the world in a raster tile format.
+
+This repository contains the documentation source files for Mapzen's Terrain Tiles:
+
+## [https://mapzen.com/documentation/terrain](https://mapzen.com/documentation/terrain)
```

Comment:
The sister project is `vector-tiles`, so I think the URL for terrain would be:

- `https://mapzen.com/documentation/terrain-tiles/`